doc/user_guide/postgresql_user_guide.md

```diff
@@ -23,16 +50,40 @@ CREATE SCHEMA ADAPTER;
 The SQL statement below creates the adapter script, defines the Java class that serves as entry point and tells the UDF framework where to find the libraries (JAR files) for Virtual Schema and database driver.
 
 ```sql
+--/
 CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
   %scriptclass com.exasol.adapter.RequestDispatcher;
-  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-9.0.5-postgresql-2.0.3.jar;
+  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-9.0.5-postgresql-2.0.4.jar;
   %jar /buckets/<BFS service>/<bucket>/postgresql-<postgresql-driver-version>.jar;
 /
 ```
 
+## Configuring ExaLoader
+
+You must configure ExaLoader to enable it to actually import or *read* data from the external database, see https://github.com/exasol/docker-db/#installing-custom-jdbc-drivers.
+
+ExaLoader expects the driver in BucketFS folder `default/drivers/jdbc`.<br />
+If you uploaded the driver for UDF to a different folder, then you need to [upload](#uploading-a-file-to-bucketfs) the driver again.
+
+
+Additionally  you need to create a file `settings.cfg` and [upload](#uploading-a-file-to-bucketfs) it to the same folder in BucketFS:
+
+```
+DRIVERNAME=POSTGRES_JDBC_DRIVER
+JAR=<jar file containing the jdbc driver>
+DRIVERMAIN=org.postgresql.Driver
+PREFIX=jdbc:postgresql:
+FETCHSIZE=100000
+INSERTSIZE=-1
```

Comment:
:+1: 

While this is theoretically relevant for all JDBC-based VS, there are configuration parameters here that depend on the dialect, so this is very useful.

Only 13 more dialects to go. :-)